ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml

```diff
@@ -770,6 +770,17 @@ tests:
     test:
     - ref: openshift-extended-test-longduration
     workflow: cucushift-installer-rehearse-aws-ipi-ovn
+- as: azure-ipi-ovn-hypershift-install-p1
+  cron: 0 1 * * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      TEST_FILTERS: ~ChkUpgrade&;~DisconnectedOnly&;HyperShiftINSTALL&
+      TEST_SCENRAIOS: Hypershift
```

Comment:
there is typo for TEST_SCENRAIOS. now it is changed to TEST_SCENARIOS

Reply:
done